src/Components/Login/Login.js

```diff
@@ -0,0 +1,72 @@
+import React, {useState} from 'react';
+import login from './login.module.scss';
+import logo1 from '../../assets/logo1.png';
+import email from '../../assets/email1.svg';
+import eyeopen from '../../assets/show.svg';
+import eyeclose from '../../assets/hide.svg';
+import lock from '../../assets/passw.svg';
+
+
+
+
+const Login = () => {
+  const [showpassword, setshowpassword]=useState(false);
+  function Toggle() {
+    var temp = document.getElementById("typepass");
+    if (temp.type === "password") {
+        temp.type = "text";
+        setshowpassword(true);
+    }
+    else {
+        temp.type = "password";
+        setshowpassword(false);
+    }
+}
+  return (
+    <div className={login.loginpage}>
+        <form className={login.form}>
+      <img src={logo1} alt='logo' className={login.logo}/>
+      <p className={login.head}>Hello, Welcome back!</p>
+      <div className={login.emailinfo} >
+        <img src={email} alt='email' className={login.emailimg}/>
+        <input
```

Comment:
take the input in useState and add function to handle that input and for now print it on console

Reply:
done